### PR TITLE
Release 27.2.0

### PR DESCRIPTION
**Pull request by jon-kirwan**
* Add new scroll tracking ([PR #2305](https://github.com/alphagov/govuk_publishing_components/pull/2305))
* Update crown fallback image ([PR #2313](https://github.com/alphagov/govuk_publishing_components/pull/2313)) PATCH
* Remove heading and use strong in step nav header ([PR #2311](https://github.com/alphagov/govuk_publishing_components/pull/2311))

### CODE REVIEW

**Comment by andysellick** [comment]
@jon-kirwan if this isn't urgent can we include https://github.com/alphagov/govuk_publishing_components/pull/2305? It's been approved, just waiting on a final comment before merging.

**Comment by jon-kirwan** [comment]
> @jon-kirwan if this isn't urgent can we include #2305? It's been approved, just waiting on a final comment before merging.

Yep no problem. Will wait until it's ready to be merged 👍

**Comment by andysellick** [comment]
@jon-kirwan https://github.com/alphagov/govuk_publishing_components/pull/2305 is merged, so you can go ahead and rebase this PR, thanks.

File: Gemfile.lock
Comment (andysellick) [inline Gemfile.lock:102]: Are these additional changes to the Gemfile.lock necessary? Normally these are handled by dependabot PRs, so gem version updates (like this) are solely about versioning the components gem itself, and nothing else.

Comment (jon-kirwan) [reply Gemfile.lock:102]: Perhaps not but I'm finding that they are bumped each time I run `bundle install && yarn install` - but it does seem odd. I can't see any reason for the versions to be changed.

Comment (andysellick) [reply Gemfile.lock:102]: The documentation does say to do that, but I think that's incorrect (I'll look at changing that). Meanwhile can I suggest that you revert these changes and only modify the `govuk_publishing_components` line in this file?

Comment (jon-kirwan) [reply Gemfile.lock]: No problem, just updated that now

**Review by andysellick — APPROVED** [review]
🚀

**Pull request merged**
